cmd/root.go

```diff
@@ -40,6 +41,12 @@ func init() {
 	if err != nil {
 		slog.Error("error while binding json flag", slog.Any("error", err))
 	}
+
+	rootCmd.PersistentFlags().BoolVarP(&AutoPadding, "auto-padding", "p", false, "Enable automatic padding of payload. (default: false)")
```

Comment:
Please add a warning that the output may get corrupted.

Reply:
<img width="582" alt="Screenshot 2024-12-04 at 14 56 37" src="https://github.com/user-attachments/assets/ea70b2ff-697f-4e25-bb77-18914151d088">
like this?

Reply:
Looks good thanks!